ocaml_emulator/riscv_ocaml_sim.ml

```diff
@@ -61,6 +65,9 @@ let options = Arg.align ([("-dump-dts",
                           ("-test-signature",
                            Arg.String set_signature_file,
                            " file for signature output (requires ELF signature symbols)");
+                          ("-signature-granularity",
+                           Arg.Int set_signature_granularity,
+                           " test signature granularity (in Bytes)");
```

Comment:
```suggestion
                           " test signature granularity (in bytes)");
```
unless Bytes specifically is used elsewhere that I can't see (I can see MB but that's different)